test/src/cookies.spec.ts

```diff
@@ -750,5 +778,87 @@ describe('Cookie specs', () => {
       // Expect the cookie for the COOKIE_DESTINATION from URL_1 is deleted.
       expect(await page.cookies(COOKIE_DESTINATION_URL)).toHaveLength(0);
     });
+    it('should only delete cookie without partition key if partitionkey is not specified', async () => {
+      const {page, server} = await getTestState();
+      const url = 'https://example.com';
+
+      await page.goto(server.EMPTY_PAGE);
+      await page.setCookie({
+        url,
+        name: 'partitionCookie',
+        value: 'partition',
+        partitionKey: 'http://localhost',
+      });
+
+      await page.setCookie({
+        url,
+        name: 'partitionCookie',
+        value: 'partition',
+      });
+
+      await page.deleteCookie({
```

Comment:
this test would not work since by default the partition key from the page is computed in setCookie and deleteCookie and it's all the same http://localhost used in tests. I think we should try changing the parition key to something else in the first setCookie

Reply:
done. changed to https://example.com.